Tasks/Src/RTOS_BPS.c

```diff
@@ -38,9 +40,19 @@ void RTOS_BPS_MutexPost(BPS_OS_MUTEX* mutex, BPS_OS_OPT options) {
     OSMutexPost(mutex, options, &err);
     assertOSError(err);
 }
-void RTOS_BPS_TaskCreate(void){
+void RTOS_BPS_TaskCreate(
+    OS_TCB        *p_tcb,
+    CPU_CHAR      *p_name,
+    OS_TASK_PTR    p_task,
+    void          *p_arg,
+    OS_PRIO        prio,
+    CPU_STK       *p_stk_base,
+    CPU_STK_SIZE   stk_size,
+    OS_ERR        *p_err){
+        OSTaskCreate(p_tcb, p_name, p_task, p_arg, prio, p_stk_base, WATERMARK_STACK_LIMIT, stk_size, 0, 10,(void *)0, OS_OPT_TASK_STK_CHK | OS_OPT_TASK_SAVE_FP, p_err);
+        assertOSError(*p_err);
+        }
 
-}
 void RTOS_BPS_TimeDelay(void){
 
 }
```

Comment:
Add a newline to the end of all files. You can look at the coding styles guide on the BPS documentation for more information.